packages/elements/src/components/ino-markdown-editor/readme.md

```diff
@@ -0,0 +1,50 @@
+# ino-markdown-editor
+
```

Comment:
We should add a description that contains the supported actions for the editor (bold, italics, code, ...).

Reply:
Description is added to `.tsx` file